lgc/patch/Patch.cpp

```diff
@@ -247,7 +247,7 @@ void Patch::addPasses(PipelineState *pipelineState, lgc::PassManager &passMgr, b
 //
 // @param [in/out] passMgr : Pass manager
 void Patch::registerPasses(lgc::PassManager &passMgr) {
-#define LLPC_PASS(NAME, CLASS) passMgr.registerPass(NAME, decltype(CLASS)::name());
+#define LLPC_PASS(NAME, CLASS) passMgr.registerPass(NAME, CLASS::name());
```

Comment:
We should do the same thing in `llpcSpirvLower.cpp` and also update `llpc/lower/PassRegistry.inc` for consistency. We also need to update the doc (`llpc/docs/AddingPasses.md`).

Reply:
You're right, I've updated the PR.